apps/studio/components/interfaces/TableGridEditor/SidePanelEditor/RowEditor/InputField.tsx

```diff
@@ -223,14 +227,16 @@ const InputField = ({
         error={errors[field.name]}
         onChange={(event: any) => onUpdateField({ [field.name]: event.target.value })}
         actions={
-          <Button
-            type="default"
-            htmlType="button"
-            onClick={() => onEditJson({ column: field.name, value: field.value })}
-            icon={<Edit2 />}
-          >
-            Edit JSON
-          </Button>
+          isEditable && (
```

Comment:
nit: thinking it'll still be good to allow users to view JSON in the side panel if this might be a quick one